tests/rbac/test/test_fixtures.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2018 Contributors to Hyperledger Sawtooth
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+"""Test the common test fixtures"""
+# pylint: disable=wildcard-import,unused-wildcard-import,redefined-outer-name
+
+import pytest
+from tests.rbac.test.fixtures import *
+from tests.rbac.test.assertions import *
+
+from rbac.common.logs import getLogger
+
+LOGGER = getLogger(__name__)
+
+
```

Comment:
We don't need tests for utilities that we leverage to feed/generate/fixture test data. That said: There's little harm in having it here other than the need to evolve it alongside the model/payload.

Reply:
Got it. An artifact of testing that the fixtures worked before actually using them. (For example, some of the test data methods aren't used yet).